core/ResManager/plResManager.h

```diff
@@ -106,6 +106,24 @@ class HSPLASMA_EXPORT plResManager
      */
     void setVer(PlasmaVer pv, bool force = false);
 
+    /**
+     * Set whether or not keyed object IDs should be preserved from
+     * PRPs that are read in.  By default, key IDs are reordered to be
+     * consecutive when a PRP is read.  This sitting doesn't have any
+     * effect on data already loaded.
+     * \param preserve Preserve keyed object IDs
+     * \sa getPreserveObjIDs()
+     */
+    void setPreserveObjIDs(bool preserve) { preserveIDs = preserve; }
```

Comment:
FWIW, the original reason I didn't expose this except in the constructor was to ensure we couldn't end up in a situation where we had files open with both preserved key IDs and resequenced key IDs in the same ResMgr session.

Reply:
Yeah, I was thinking that as well - but I noticed that `setVer` already had a similar disclaimer, so I felt this was OK.... especially considering it would require re-allocating the res manager it `pyResManager__init__()` if this function didn't exist.